### PR TITLE
Final commit

### PR DESCRIPTION
**Pull request by CharlesSquirel**
Final commit, challenge done.

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
Develop a website navigation section with hover effect displaying pictures.

### Requirements
1. Section should have 6 interactive text elements.
2. Each element should change color on hover.
3. Each element should have a different picture displayed while hovering.
4. Each picture displayed while hovering should have a different placement.

### Code Changes Review

#### File: challenge-1/styles/_hovers.scss
```diff
@@ -9,7 +9,7 @@
   content: url("../assets/interiors.jpg");
   top: 50%;
   transform: translateY(-50%);
-  right: -420px;
+  right: -419px;
 }
```

#### File: challenge-1/styles/style.css
```diff
@@ -26,7 +26,7 @@ body {
   content: url("../assets/interiors.jpg");
   top: 50%;
   transform: translateY(-50%);
-  right: -420px;
+  right: -419px;
 }
```

The provided code changes only show a modification in the placement of a picture that is displayed when hovering over an element. The change is from `right: -420px;` to `right: -419px;` in both `_hovers.scss` and `style.css` files.

Based on the provided code changes, it's not possible to validate if all task requirements are met:

1. The code changes do not show whether there are 6 interactive text elements.
2. The code changes do not show whether each element changes color on hover.
3. The code changes do not show whether each element has a different picture displayed while hovering. Only one picture (`interiors.jpg`) is shown in the changes.
4. Although the code changes show a different placement for a picture, it's not clear if each picture displayed while hovering has a different placement as only one picture placement is shown in the changes.

In conclusion, the provided code changes do not provide enough information to validate if all task requirements are met. More comprehensive code changes are needed to fully assess the implementation against the task requirements.
